frontends/p4/sideEffects.h

```diff
@@ -128,6 +128,14 @@ class DoSimplifyExpressions : public Transform, P4WriteContext {
     const IR::Expression* addAssignment(Util::SourceInfo srcInfo, cstring varName,
                                         const IR::Expression* expression);
     bool mayAlias(const IR::Expression* left, const IR::Expression* right) const;
+    const IR::Parameter* findAlias(const IR::Parameter* p,
```

Comment:
please document this function.  In general expr1 seeems to be a subexpression of an argument assigned to p.